samples/bot-tag-mention/csharp/README.md

```diff
@@ -81,7 +81,7 @@ This sample app demonstrates the use of tag mention funtionality in teams scope
    - Select Configuration section.
    - Under configuration -> Add OAuth connection string.
    - Provide connection Name : for eg `ssoconnection`
-   - Select service provider ad `Azure Active Directory V2`
+   - Select service provider ad `Microsoft Entra ID V2`
```

Comment:
Undo this it should be Azure Active Directory

Reply:
fixed